docs/pages/usekshow.vue

```diff
@@ -0,0 +1,196 @@
+    </DocsPageSection>
+
+    <DocsPageSection title="Example" anchor="#example">
+      <p>This is a simulation of a typical use-case of showing a loader while fetching data. You can set your own fetch request length and minimum visible time, and then hit the fetch button to see the output.</p>
+
+      <DocsShowCode language="html">
+        <KTransition kind="component-fade-out-in">
+          <KCircularLoader
+            v-if="show('key-1', isFetching, minVisibleTime )"  
+          />
+          <div v-else>
+            Loaded!
+          </div>
+        </KTransition>
+      </DocsShowCode>
+
+      <div :style="{ marginTop: '24px', display: 'flex' }">
+        <div>
+          <span :style="{ marginLeft: '8px' }">Output:</span>
+          <DocsShow>
+            <div :style="{ width: '200px', height: '160px', display: 'flex', flexDirection: 'column', justifyContent: 'space-between', padding: '12px 2px 4px 2px' }">
+              <div>
+                <KTransition kind="component-fade-out-in">
+                  <KCircularLoader
+                    v-if="show('key-1', isFetching, minVisibleTime )"
+                    key="loader"
+                    disableDefaultTransition
+                  />
+                  <div v-else key="message" :style="{ textAlign: 'center' }">
+                    Loaded!
+                  </div>
+                </KTransition>
+              </div>
+              <div>
+                <code>isFetching: {{ isFetching }}</code>
+                <code>minVisibleTime: {{ minVisibleTime }}</code>
+              </div>
+            </div>
+          </DocsShow>
+        </div>
+        <div :style="{ marginTop: '28px', marginLeft: '24px' }">
+          <span>
+            <label :style="{ display: 'block' }">Fetch request length (ms)</label>
+            <input
+              v-model="fetchingTimeInput"
+              type="number"
+              :style="{ display: 'block' }"
+            >  
+          </span>
+          <span>
+            <label :style="{ display: 'block', marginTop: '12px' }">
+              Minimum visible time (ms)
+            </label>
+            <input
+              v-model="minVisibleTimeInput"
+              type="number"
+              :style="{ display: 'block' }"
+            >  
+          </span>
+          <KButton :style="{ marginTop: '24px' }" @click="fetchData">
+            Fetch data
+          </KButton>
+        </div>
+      </div>
+    </DocsPageSection>
+
+    <DocsPageSection title="Related" anchor="#related">
+      <ul>
+        <li>
+          Some components offer a simpler interfance to achieve the same effect when there is no need to be switching between more components. For example, see <DocsInternalLink href="/kcircularloader#prop:minVisibleTime">
+            KCircularLoader's <code>minVisibleTime</code>
+          </DocsInternalLink>.
+        </li>
+      </ul>
+    </DocsPageSection>
+
+    <DocsPageSection title="Parameters" anchor="#parameters">
```

Comment:
"Parameters" and "Returns" would ideally get automatically generated from JSDoc comments of the composable function, similarly how "Props" are generated from components. I believe this would be helpful for the other two composables we have in KDS already. Will open an issue for this as soon as review of this PR is done.